local-dev/src/components/GlobalStatus.js

```diff
@@ -48,15 +45,51 @@ class GlobalStatus extends Component {
             default:
                 break;
         }
+        return img;
+    }
+    setFavicon(status) {
+        // update favicon
+        var favicon = document.getElementById('favicon');
+        var img = this.statusToIcon(status);
 
         if (img) {
             favicon.href = img;
         }
     }
+    realDesktopNotify(message, icon) {
+        var options = {
+            body: 'Mozilla Engagement Engineering Status Board',
+            icon: icon
+        }
+        new Notification(message, options);
+    }
+    desktopNotify(message, icon) {
+        if (!('Notification' in window)) {
+            return;
+        }
```

Comment:
Since this function is called repeatedly, could we do the feature detection once and reference the result either here or in the `render` function below?